core/src/utils/test/playwright/page/utils/set-content.ts

```diff
@@ -19,25 +19,30 @@ export const setContent = async (page: Page, html: string, testInfo: TestInfo, o
 
   let mode: Mode;
   let direction: Direction;
+  let theme: Theme;
 
   if (options == undefined) {
     mode = testInfo.project.metadata.mode;
     direction = testInfo.project.metadata.rtl ? 'rtl' : 'ltr';
+    theme = testInfo.project.metadata.theme;
   } else {
     mode = options.mode;
     direction = options.direction;
+    theme = options.theme;
   }
 
   const baseUrl = process.env.PLAYWRIGHT_TEST_BASE_URL;
 
   const output = `
     <!DOCTYPE html>
-    <html dir="${direction}">
+    <html dir="${direction}" lang="en">
       <head>
+        <title>Ionic Playwright Test</title>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
         <link href="${baseUrl}/css/ionic.bundle.css" rel="stylesheet" />
         <link href="${baseUrl}/scripts/testing/styles.css" rel="stylesheet" />
+        <link href="${baseUrl}/scripts/testing/themes/${theme}.css" rel="stylesheet" />
```

Comment:
`theme` is currently always defined in this capacity since we assign a default of `light` when the test config doesn't specify an explicit theme to test against. 

I'll rework this a bit so that `default` is an available option for the fallback state. If the theme is `default` it will opt-out of importing a theme stylesheet.

Reply:
Addressed here: https://github.com/ionic-team/ionic-framework/pull/28593/commits/54289d8757cb579a37d823172fab74c04190dfcc (both feedback comment)